### PR TITLE
Fixes for issue86

### PR DESCRIPTION
**Pull request by TimHenkelmann**
Fixed all issues that were reported in #86

### CODE REVIEW

**Review by kordianbruck — APPROVED** [review]
This makes sense - why not. (Not an expert tho)

File: cli.js [hunk 1]
```diff
@@ -152,6 +153,15 @@ function populateDB(postgresConnectionString) {
              4. dbpedia works/releases (as array in the artists input)
              */
 
+            //prepopulate instrumentsArray
+            const instruments = context.models.instruments;
+            instruments.findAll().then((queriedInstruments)=>{
```
Comment (sacdallago) [inline cli.js:158]: This will become slow if the amount of instruments is real high. Though, it _works_, for now

File: cli.js [hunk 2]
```diff
@@ -339,6 +356,10 @@ function dbpediaPopulateArtists(context) {
 }
 
 function connectArtistToWorks(context, createdArtist, work) {
+    if(work.startsWith("dbr:")){
```
Comment (sacdallago) [inline cli.js:359]: nice to see some cleaning of the data!

**Review by sacdallago — COMMENTED** [review]
Too late anyway: but ok

**Pull request merged**